inst/example_apps/rmd_dashboard_app/app.R

```diff
@@ -1,15 +1,22 @@
-
-library(shiny)
 library(crunchy)
-# Render the dashboard
-rmarkdown::render("dashboard.Rmd")
+
 ui <- fluidPage(
-    includeHTML("dashboard.html")
+     uiOutput("dashboard")
 )
 
-server <- function(input, output) {
+server <- function(input, output, session) {
+    user <- shinyUser()
+    output$dashboard <- renderUI({
+        current_user <- email(user())
```

Comment:
To use `email` and not `crunch:::email`, add `Crunch-io/rcrunch` to `Remotes:` in DESCRIPTION for now (and be sure to tell Kyle of that requirement to run the examples)